v19.2/experimental-features.md

```diff
@@ -141,19 +140,9 @@ The table below lists the experimental SQL functions and operators available in
 | [`experimental_strptime`](functions-and-operators.html#date-and-time-functions)  | Format time using standard `strptime` notation. |
 | [`experimental_uuid_v4()`](functions-and-operators.html#id-generation-functions) | Return a UUID.                                  |
 
-## Vectorized query execution
+## Vectorized execution on non-streaming operations
 
-<span class="version-tag">New in v19.2</span>: When the `experimental_vectorize` setting is enabled, vectorized query execution uses a columnar orientation to dramatically increase query execution performance for supported OLTP Heavy/[Online analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing) queries. By default, CockroachDB uses a SQL execution engine organized in a row orientation, which offers good performance for [Online transaction processing (OLTP)](https://en.wikipedia.org/wiki/Online_transaction_processing) queries but can offer suboptimal performance for OLTP Heavy/OLAP queries.
-
-For an example of how vectorized query execution works on vectorizing a merge join, see the blog post [Vectorizing the merge joiner in CockroachDB](https://www.cockroachlabs.com/blog/vectorizing-the-merge-joiner-in-cockroachdb/).
-
-{{site.data.alerts.callout_danger}}
-The vectorized execution engine has the following limitations:
-
-- It does not yet monitor memory usage or spill to disk for large joins, so there is a risk that queries that perform large joins or other memory-intensive operations could cause the node to run out of memory and crash.
-
-- It is undertested compared to CockroachDB's existing row-oriented execution engine.
-{{site.data.alerts.end}}
+[Vectorized query execution](vectorized-execution.html) in CockroachDB is experimental for non-streaming operations, including global sorts, hash joins, merge joins on non-unique columns, and window functions. By default, CockroachDB uses vectorized execution on streaming operations. You can turn vectorized execution on for all operations by setting the `sql.defaults.vectorize` [cluster setting](cluster-settings.html) or the `vectorize` [session variable](set-vars.html) to `experimental_on`.
```

Comment:
I found this paragraph hard to read as someone who doesn't already know stuff about this feature.  Maybe it would be clearer as something like "By default, CockroachDB does not support vectorized execution for the following non-streaming operations: (bulleted list with line breaks).  To turn on experimental support in the vectorized execution engine for non-streaming operations, set (bulleted list of settings flags with line breaks)."  I would also like the streaming operations to link to the things they are talking about, e.g., merge joins link to merge join section of JOIN docs, etc.  In general the whole "streaming" vs "non-streaming" operations makes a lot of sense if you think about DistSQL all day, but it probably won't to many normal users (including me unless I need to care), so we should be way more explicit about what things go in what categories.